### PR TITLE
[JS-loaders] add other composites to loaders

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Adds other composites to the JS loaders package
# Why

allows contoso in alpha to use all the composites from the TS API

https://skype.visualstudio.com/SPOOL/_workitems/edit/3785864
# How Tested

Build local version and tested API in Angular app 
image

### CODE REVIEW

File: packages/acs-ui-javascript-loaders/src/callWithChatCompositeLoader.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+const reactVersion = React.version;
+const parseReactVersion = (version: string | undefined): number[] => {
+  if (!version) {
+    return [];
+  }
+  return version.split('.').map((v) => parseInt(v));
+};
+if (parseReactVersion(reactVersion)[0] && parseReactVersion(reactVersion)[0] < 18) {
+  throw new Error(
```
Comment (emlynmac) [inline packages/acs-ui-javascript-loaders/src/callWithChatCompositeLoader.ts:14]: Should this be pulled out to a common function?

File: packages/acs-ui-javascript-loaders/src/chatCompositeLoader.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+
+const reactVersion = React.version;
+const parseReactVersion = (version: string | undefined): number[] => {
```
Comment (emlynmac) [inline packages/acs-ui-javascript-loaders/src/chatCompositeLoader.ts:7]: Another copy.

File: packages/acs-ui-javascript-loaders/src/utils.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * @private
+ */
+export const parseReactVersion = (version: string | undefined): void => {
+  if (!version) {
+    return;
+  }
+  const reactVersion = version.split('.').map((v) => parseInt(v));
+  if (reactVersion[0] && reactVersion[0] < 18) {
+    throw new Error(
+      'React version is less than 18. Please upgrade to React 18 or alternatively checkout how to use our composites directly here: https://azure.github.io/communication-ui-library/?path=/docs/quickstarts-composites--page'
```
Comment (emlynmac) [inline packages/acs-ui-javascript-loaders/src/utils.ts:16]: We should probably get this behind an aka.ms url, for the upcoming storybook 8 switchover.

**Review by emlynmac — APPROVED** [review]

File: packages/acs-ui-javascript-loaders/src/utils.ts (hunk 3 above)
Comment (dmceachernmsft) [reply packages/acs-ui-javascript-loaders/src/utils.ts:16]: are we going to reassign the storybook aka link to v8? or is there going to be a new aka link while we are vetting it?

Comment (emlynmac) [reply packages/acs-ui-javascript-loaders/src/utils.ts:16]: The link could be updated when we switch; so this code won't need updating at that time. If it's left this way, we'll need to keep note that this URL will need to updated potentially.

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**